src/test/pretty/fn-variadic.rs

```diff
@@ -0,0 +1,15 @@
+// Check that `fn foo(x: i32, ...)` does not print as `fn foo(x: i32, ..., ...)`.
+// See issue #58853.
+
+// pp-exact
+#![feature(c_variadic)]
+
+extern "C" {
+    pub fn foo(x: i32, ...);
+}
+
+pub unsafe extern "C" fn bar(_: i32, mut ap: ...) -> usize {
+    ap.arg::<usize>()
+}
+
+fn main() { }
```

Comment:
@alexreg looks like this has to be formatted as `fn main() { }`

Reply:
Oh really? Sorry, my bad. I'm so used to `{}` in other files, I thought it was actually standard.